index.html

```diff
@@ -12,20 +12,22 @@
 
 <body>
     <!-- Header start -->
-    <header>
-        <div>
-            <div>
-                <img src="" alt="logo">
-            </div>
-            <div>
-                <a href="#">home</a>
-                <a href="#">pricing</a>
-                <a href="#">career</a>
-                <a href="#">pages</a>
-                <a href="#">blog</a>
-                <a href="#">contact</a>
-                <a href="#">login</a>
-                <a href="#">join now</a>
+    <header class="container">
+        <div class="col-12">
+            <div class="row">
+                <div class="col-lg-2 col-sm-4 col-8">
+                    <img class="logo" src="#" alt="logo DocLand">
+                </div>
+                <div class="col-lg-10 col-sm-8 col-4">
```

Comment:
![image](https://user-images.githubusercontent.com/78425010/119723030-ae521c80-be75-11eb-91b4-67fbb97b0f3a.png)
floatinasi į dešinę su klase: 'reverse-lg' (arba kitais dydžiais)